src/tag.rs

```diff
@@ -1472,8 +1472,15 @@ impl<'a> Tag {
     }
 
     /// Reads AIFF file and returns ID3 Tag from it
-    pub fn read_from_aiff(path: impl AsRef<Path>) -> crate::Result<Tag> {
-        aiff::load_aiff_id3(path)
+    pub fn read_from_aiff_file(path: impl AsRef<Path>) -> crate::Result<Tag> {
+        let file = File::open(path)?;
+        let mut reader = BufReader::new(file);
+        aiff::load_aiff_id3(&mut reader)
+    }
+
+    /// Read ID3 tag from AIFF data in reader
+    pub fn read_from_aiff(reader: &mut (impl io::Read + io::Seek)) -> crate::Result<Tag> {
```

Comment:
Shall we call this function `read_from_aiff_stream`? Also, the argument type is rather complex. Could we just use `reader: impl io::Read + io::Seek` instead?

Reply:
It throws error that it has to be mutable. `cannot borrow as mutable`

Reply:
Sooo, `mut reader`?

Reply:
Sorry, relatively new to Rust, didn't know the syntax was that way too. Added in new commit. Thank you.

Reply:
:+1: